motion/ruby_motion_query/device.rb

```diff
@@ -92,7 +92,7 @@ def iphone?
       def simulator?
         if @_simulator.nil?
           @_simulator = if ios_at_least?(9.0)
-            !NSBundle.mainBundle.bundlePath.start_with?('/var/')
+            !(NSBundle.mainBundle.bundlePath =~ /Library\/Developer/i).nil?
```

Comment:
Sorry I see now: `UIDevice.currentDevice.model` versus `UIDevice.currentDevice.name`

do you want to update your PR to use the name to compare instead of the bundlePath stuff we are using here?